--- origsrc/lighttpd-1.4.19/src/mod_webdav.c	2007-07-17 13:02:33.000000000 -0500
+++ src/lighttpd-1.4.19/src/mod_webdav.c	2008-08-08 00:49:30.921875000 -0500
@@ -23,7 +23,7 @@
 #include <sqlite3.h>
 #endif
 
-#if defined(HAVE_LIBXML_H) && defined(HAVE_SQLITE3_H) && defined(HAVE_UUID_UUID_H)
+#if defined(HAVE_LIBXML_H) && defined(HAVE_SQLITE3_H) && defined(HAVE_UUID)
 #define USE_LOCKS
 #include <uuid/uuid.h>
 #endif
