--- origsrc/lighttpd-1.4.19/src/Makefile.am	2008-02-26 10:21:09.000000000 -0600
+++ src/lighttpd-1.4.19/src/Makefile.am	2008-08-08 00:45:05.984375000 -0500
@@ -16,8 +16,8 @@
 configparser.c configparser.h:
 mod_ssi_exprparser.c mod_ssi_exprparser.h:
 else
-$(srcdir)/configparser.y: lemon
-$(srcdir)/mod_ssi_exprparser.y: lemon
+$(srcdir)/configparser.y: lemon$(EXEEXT)
+$(srcdir)/mod_ssi_exprparser.y: lemon$(EXEEXT)
 
 configparser.c configparser.h: $(srcdir)/configparser.y
 	rm -f configparser.h
